docs/docs/whatsnew.md

```diff
@@ -1,5 +1,30 @@
 # <b>Important updates</b>
 
+## v1.2.25
+
+### Configuration of `maven` builder has changed. 
+As of version 1.2.25, the `mvn -B clean package` command is used where the `maven` builder is configured for building a module or in a global build step. 
+
+By adding the -B parameter, the "maven clean package" command will start in interactive mode.
+
+As a build tool, MBT builds the MTA (Multi-Target Application), which contains many type of modules. Each module is built by corresponding builder, such as maven, nodejs and go.
```

Comment:
such as maven, node.js, and go.
(Doesn't it need to have the period in node.js? Also, when I looked up these builders, they were all listed with capital first letter- Maven, Node.js, and Go. Is that required here?)

Reply:
I think maven, npm and golang is better for native builder name.

As to the builder name, you can reference : https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml

In the above doc, all builder's name is low case， maven is for java type module, golang for go type module, npm for nodejs type module